pephubclient/samples/samples.py

```diff
@@ -0,0 +1,38 @@
+from ..files_manager import FilesManager
+
+
+class Samples:
```

Comment:
Is this an object that represents multiple `Samples` ?

Or is the plurality here from SQLmodel since the table name is samples? it's a bit confusing, maybe explain docstring what this holds?